tests/test_crs.py

```diff
@@ -200,15 +200,6 @@ def test_equality_from_dict(epsg_code):
 
 
 def test_is_same_crs():
-    crs1 = CRS({'init': 'epsg:4326'})
-    crs2 = CRS({'init': 'epsg:3857'})
-
-    assert crs1 == crs1
-    assert crs1 != crs2
-
-    wgs84_crs = CRS.from_string('+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs')
-    assert crs1 == wgs84_crs
```

Comment:
I'm removing this test as it can't succeed anymore. The slow `to_epsg()` checks that we used to make in `CRS.__eq__()` covered up GDAL/PROJ changes, but we are no longer covering them up. This only affects people who are using the obsolete `+init=epsg:dddd` parameter.